docs/developers/index.md

```diff
@@ -32,6 +32,15 @@ Need a more basic primer first? Check out [ethereum.org/learn.](/learn/)
 
 Any program that runs on the Ethereum Virtual Machine (EVM) is commonly referred to as a “smart contract”. The most popular languages for writing smart contracts on Ethereum are **Solidity** and **Vyper**, though there are [others under development](https://github.com/ConsenSys/ethereum-developer-tools-list#smart-contract-languages).
 
+## Language Specific Landing Pages
+
+We're building a suite of language-specific landing pages for developer to learn about Ethereum in their preferred programming langauge.
+
+- [Java ♡ Ethereum](/java)
```

Comment:
I like the hearts but think it'd be more descriptive to state "Ethereum for Java developers", "Ethereum for Javascript developers", etc.

Reply:
Let's also update this link to `/developers/java` - reasoning in my comment on docs/java/index.md

Reply:
Fixed the header